packages/salesforcedx-vscode-core/src/telemetry/telemetry.ts

```diff
@@ -164,6 +164,19 @@ export class TelemetryService {
     }
   }
 
+  public sendMetadataTypes(
```

Comment:
Should this be a more generic method that we use to log things that are not errors or command executions ? It could be something like `sendEventData` or something like that.

Reply:
Yeah, I think that's a good idea!